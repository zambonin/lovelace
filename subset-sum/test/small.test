#!/bin/sh

./subset-sum/ss 10 53 32 10
