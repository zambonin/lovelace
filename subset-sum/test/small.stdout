Number of subsets = 3
